### PR TITLE
[SDPA-2747] Event page e2e test

### PR DESCRIPTION
**Pull request by alan-cole**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-2747

### Changed

* Added e2e test for event page.
* Added addWysiwygMediaEmbed() for adding a media (e.g. image) to a wysiwyg.

### Screenshots

Screen Shot 2019-07-12 at 4 06 31 pm

### CODE REVIEW

File: packages/ripple-test-tools/page-models.js [hunk 1]
```diff
@@ -133,9 +133,40 @@ const adminPageModels = {
...
+    eventAuthor: {
+      fullName: dataSel('edit-field-node-author-0-value'),
+      emailAddress: dataSel('edit-field-node-email-0-value'),
+      contactPhone: dataSel('edit-field-node-phone-0-value')
+    }
+  },
   jwt: {
-    expiry: dataSel('edit-jwt-exp'),
-    key: dataSel('edit-jwt-key')
+    expiry: dataSel('#edit-jwt-exp'),
```
Comment (dylankelly) [inline packages/ripple-test-tools/page-models.js]: I dont think this is right - was the hash added by mistake?

File: test/e2e/integration/Events/Events.feature [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  Scenario: Event Page loads correctly
+    When I have navigated to the created page
+    Then the page title should be "2-BE-event-1"
+    And the Description should be "Now Neptune had gone off to the Ethiopians, who are at the world's end, and lie in two halves, the one looking West and the other East. He had gone there to accept a hecatomb of sheep and oxen, and was enjoying himself at his festival; but the other gods met in the house of Olympian Jove, and the sire of gods and men spoke first."
+    And the Dates should be "20 May 2019 11:02 am - 13 May 2020 11:11 am"
+    And the Address should be "45 Collins St,  Melbourne, VIC 3000"
+    And the Price should be "34.33 - 89.95"
+    And the Web url should be "https://www.weatherzone.com.au/vic/central/healesville"
+    And the Requirements the should be "Accessible venue"
```
Comment (dylankelly) [inline test/e2e/integration/Events/Events.feature]: And the Requirements ~the~ should be "Accessible venue"

Comment (alan-cole) [reply test/e2e/integration/Events/Events.feature]: Fixed.

File: packages/ripple-test-tools/page-models.js (hunk 1 above)
Comment (alan-cole) [reply packages/ripple-test-tools/page-models.js]: Yeah, must have been an overzealous multi-cursor.

**Pull request merged**